scripts/lldb/.lldb/o3destd_lldb.py

```diff
@@ -0,0 +1,307 @@
+        short_string_storage = storage_element.GetChildMemberWithName("m_shortData")
+
+        # Check if the Short String optimization is active
+        is_sso_active_member = short_string_storage.GetChildMemberWithName("m_packed").GetChildMemberWithName("m_ssoActive")
+        is_sso_active = int(is_sso_active_member.GetValueAsUnsigned())
+        if is_sso_active:
+            # Read the string size and memory address
+            string_size = int(short_string_storage.GetChildMemberWithName("m_packed").GetChildMemberWithName("m_size").GetValueAsUnsigned())
+            string_address = short_string_storage.GetChildMemberWithName('m_buffer').AddressOf()
+        else:
+            # Short String optimization is not active
+            try:
+                allocated_storage = storage_element.EvaluateExpression("m_allocatedData")
+            except Exception as e:
+                return f'error: {e}'
+
+            string_size = allocated_storage.GetChildMemberWithName("m_size").GetValueAsUnsigned()
+            string_address = allocated_storage.GetChildMemberWithName("m_data")
+
+        # If the string size is 0, then return an empty string
+        if string_size == 0:
+            return '""'
+
+        # Stores the result of summary string
+        result_string = ''
+        try:
+            string_type = valobj.GetType()
+            element_type = string_type.GetTemplateArgumentType(0)
+            error = lldb.SBError()
+
+            # SBType.GetBasicType is strange, passing in the enum value of the type
+            # doesn't query the actual type but returns a new SBType
+            # instance represents the basic type
+            # https://lldb.llvm.org/python_api/lldb.SBType.html#lldb.SBType.GetBasicType
+            if element_type == element_type.GetBasicType(lldb.eBasicTypeChar):
+            # Get the byte size of the character type being stored
+                element_size = element_type.GetByteSize()
+                byte_data = process.ReadMemory(string_address.GetValueAsUnsigned(), string_size * element_size, error)
+                result_string = byte_data.decode('utf-8')
+            elif element_type == element_type.GetBasicType(lldb.eBasicTypeWChar):
+                element_size = element_type.GetByteSize()
+                if element_size == 4:
+                    byte_data = process.ReadMemory(string_address.GetValueAsUnsigned(), string_size * element_size, error)
+                    result_string = byte_data.decode('utf-32')
+                if element_size == 2:
+                    byte_data = process.ReadMemory(string_address.GetValueAsUnsigned(), string_size * element_size, error)
+                    result_string = byte_data.decode('utf-16')
+            else:
+                # element type is not handled(not a char or wchar_t)
+                return '"<not handled>"'
+
+            if error.Fail():
+                return f'process read error: {error}'
```

Comment:
nit: use `<`, '>' to represent an error like how `<not handled>` is done
```suggestion
                return f'<process read error: {error}>'
```